installer/lib/mix/tasks/phx.new.ex

```diff
@@ -266,8 +266,14 @@ defmodule Mix.Tasks.Phx.New do
     Code.ensure_loaded?(Hex)
   end
 
-  defp rebar_available? do
-    Mix.Rebar.rebar_cmd(:rebar3)
+  if Version.match?(System.version(), "~> 1.18") do
+    defp rebar_available? do
+      Mix.Rebar.rebar_args(:rebar3, [])
```

Comment:
Rebar is automatically installed on more recent Elixir versions, so we are good. :)

```suggestion
      true
```